landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Apache Airflow 1.10.8 & 1.10.9"
+linkTitle: "Apache Airflow 1.10.8 & 1.10.9"
+author: "Kaxil Naik"
+twitter: "kaxil"
+github: "kaxil"
+linkedin: "kaxil"
+description: "We are happy to present the new 1.10.8 and 1.10.9 releases of Apache Airflow."
+tags: ["release"]
+date: 2020-02-23
+---
+
+We are happy to present the new 1.10.8 and 1.10.9 releases of Apache Airflow.
+
+Airflow 1.10.8 contains 160 commits since 1.10.7 and includes 4 new features, 42 improvements, 36 bug fixes, and several doc changes.
+
+We released 1.10.9 on the same day as one of the Flask dependencies (Werkzeug) released 1.0 which broke Airflow 1.10.8.
+
+**Details**:
+* **PyPI**: [https://pypi.org/project/apache-airflow/1.10.9/](https://pypi.org/project/apache-airflow/1.10.9/)
+* **Docs**: [https://airflow.apache.org/docs/1.10.9/](https://airflow.apache.org/docs/1.10.9/)
+* **Changelog (1.10.8)**: [http://airflow.apache.org/docs/1.10.8/changelog.html#airflow-1-10-8-2020-01-07](http://airflow.apache.org/docs/1.10.8/changelog.html#airflow-1-10-8-2020-01-07)
+* **Changelog (1.10.9)**: [http://airflow.apache.org/docs/1.10.9/changelog.html#airflow-1-10-9-2020-02-10](http://airflow.apache.org/docs/1.10.9/changelog.html#airflow-1-10-9-2020-02-10)
+
+Some of the noteworthy new features (user-facing) are:
+
+- [Add tags to DAGs and use it for filtering in the UI](https://github.com/apache/airflow/pull/6489)
+- [New Executor: DebugExecutor for Local debugging from your IDE](http://airflow.apache.org/docs/1.10.9/executor/debug.html)
+- [Allow passing conf in "Add DAG Run" view](https://github.com/apache/airflow/pull/7281)
```

Comment:
Is this Add DAG run or is it Trigger dag run?

Reply:
On Trigger Dag Run  yes but the view is called "Add Dag Run":

![image](https://user-images.githubusercontent.com/8811558/75120300-811b2b00-5682-11ea-9108-403cfcb553db.png)

Reply:
What's the tooltip for the action it's accessible from? Cos that whole screen is new.

Reply:
![image](https://user-images.githubusercontent.com/8811558/75120832-7c587600-5686-11ea-8f48-7033b055a48c.png)

![image](https://user-images.githubusercontent.com/8811558/75120840-837f8400-5686-11ea-9d0e-791c9774e7e2.png)

Reply:
"Browse" -> "DAG Runs" -> "Add a new Record" (+ plus sign) brings this screen